### PR TITLE
74 profile page styling

### LINKED ISSUE #74: Profile Page Styling
## 📝 Task Description 

Quick ticket -- match the profile front-end to the Figma.

Screenshot 2024-08-25 at 11 56 06 PM
--> current    

Screenshot 2024-08-25 at 11 56 06 PM
-->  Figma

View the [Figma](https://www.figma.com/design/pxgrEsnOJneqsaWWeEnQoY/FreeBites?node-id=5934-8368&t=fl9QYpo0e8MjRRaO-4). 

## ✅ Acceptance Criteria 

- [ ] Does the admin frontend match the Figma?

## ✨ Resources ✨
- [How to CSS](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics)
- [Clarence's Styling Presentation](https://docs.google.com/presentation/d/10k0kGCUdR3twBIv9yDJ0hOQZCDFyoy99SGMMBguShFg/edit#slide=id.g24e07eed7b0_1_75)

### PR DESCRIPTION
**Pull request by ManuP6789**
# Description

Fixed profile section styling to match the Figma.

## Issues

Resolve #74 

## Screenshots

<div style="display: flex; justify-content: space-between;">
  
  361300701-01d0ad78-9cd5-4448-9180-0a172e3b103e
  361300551-ac92f7d9-175c-41dd-9f6c-48b622a2e2fe
</div>

## Test

You can edit your userName, firstName, and LastName to check that it is properly displayed.
Open all the profile tabs to make sure they work!

## Possible Downsides

Need to:
- Fix Profile Editing section with some minor bugs in Pronouns and checkmark redirecting you back to profile section. 
- Add live updates for profile section and edit profile section.
- Limit firstName, LastName, and UserName character count so we don't have issues with overflow.

## Additional Documentations
N/A

### CODE REVIEW

**Review by LordofMankid — APPROVED** [review]
LGTM! you can merge this in. edit profile section should be a new ticket though it's pretty janky as you mentioned

File: src/assets/icons/editbutton.png
[the remark this answers is not in the document]
Comment (ManuP6789) [reply src/assets/icons/editbutton.png:1]: yea gotchu

File: .env
Comment (johnny-t06) [inline .env:1]: Revert the env changes before merging 👍

**Review by johnny-t06 — APPROVED** [review]
LGTM, thanks guys

**Pull request merged**